CONTRIBUTING.rst

```diff
@@ -136,6 +136,9 @@ Commits
 Each commit should reference the Tor Project Trac ticket (example: ``#12345``)
 and possibly the bugfix version.
 
+From version 1.0.3, the commit title should start by ``new`` or ``fix`` or
+``change``. The commit message should contain ``Closes: #bugnumber``.
```

Comment:
A git commit hook could help you enforce this rule.
But I think that should be a different ticket.

Reply:
ok, created #29582